test/contracts/common/depositable_delegate_proxy.js

```diff
@@ -0,0 +1,172 @@
+const { toChecksumAddress } = require('web3-utils')
+const { assertAmountOfEvents, assertEvent } = require('../../helpers/assertEvent')(web3)
+const { decodeEventsOfType } = require('../../helpers/decodeEvent')
+const { assertRevert, assertOutOfGas } = require('../../helpers/assertThrow')
+const { getBalance } = require('../../helpers/web3')
+
+// Mocks
+const DepositableDelegateProxyMock = artifacts.require('DepositableDelegateProxyMock')
+const EthSender = artifacts.require('EthSender')
+const ProxyTargetWithoutFallback = artifacts.require('ProxyTargetWithoutFallback')
+const ProxyTargetWithFallback = artifacts.require('ProxyTargetWithFallback')
+
+const TX_BASE_GAS = 21000
+const SEND_ETH_GAS = TX_BASE_GAS + 9999 // 10k gas is the threshold for depositing
```

Comment:
Should we also change [`recovery_to_vault`'s limit](https://github.com/aragon/aragonOS/blob/19756078c111e3ee4a7d06aab39a7ceacc919c61/test/contracts/apps/recovery_to_vault.js#L23) to 30999 (and related values in `kernel_funds` and `app_funds`)?